.jenkins/actions/build_bare_metal.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# stop on all errors
+set -e
+
+INSTALL_DIR=${PROJECT}/../install
+FV3GFSEXE_DIR=${INSTALL_DIR}/fv3gfs-fortran/
+FV3CONFIG_VENV=${INSTALL_DIR}/venv/vcm_1.0/
+
+##################################################
+# functions
+##################################################
+
+exitError()
+{
+    echo "ERROR $1: $3" 1>&2
+    echo "ERROR     LOCATION=$0" 1>&2
+    echo "ERROR     LINE=$2" 1>&2
+    exit $1
+}
+
+containsElement()
+{
+  local e
+  for e in "${@:2}"; do [[ "$e" == "$1" ]] && return 0; done
+  return 1
+}
+
+showUsage()
+{
+    echo "usage: `basename $0` [-h]"
+    echo ""
+    echo "optional arguments:"
+    echo "-h             show this help message and exit"
+    echo "-c <compiler>  compiler to use [gnu|intel]"
+}
+
+parseOptions()
+{
+    # process command line options
+    while getopts "hc:" opt
+    do
+        case $opt in
+        h) showUsage; exit 0 ;;
+        c) compiler=$OPTARG ;;
+        \?) showUsage; exitError 301 ${LINENO} "invalid command line option (-${OPTARG})" ;;
+        :) showUsage; exitError 302 ${LINENO} "command line option (-${OPTARG}) requires argument" ;;
+        esac
+    done
+    test -n "${compiler}" || exitError 310 ${LINENO} "Option <compiler> is not set"
+    local compilers=(gnu intel)
+    containsElement "${compiler}" "${compilers[@]}" || exitError 312 ${LINENO} "Invalid compiler (${compiler}) chosen"
+}
+
+# echo basic setup
+echo "####### executing: $0 $* (PID=$$ HOST=$HOSTNAME TIME=`date '+%D %H:%M:%S'`)"
+
+# start timer
+T="$(date +%s)"
+
+# parse command line options (pass all of them to function)
+parseOptions $*
+
+# check presence of env directory
+pushd `dirname $0` > /dev/null
+envloc=`/bin/pwd`/..
+popd > /dev/null
+
+# determine root directory
+pushd ${envloc}/.. > /dev/null
+rootdir=`/bin/pwd`
+popd > /dev/null
+
+# setup module environment and default queue
+# note: disable error checking since script uses fails for checks
+set +e
+. ${envloc}/env/machineEnvironment.sh
+set -e
+
+# load module tools
+. ${envloc}/env/moduleTools.sh
+
+# load git tools
+. ${envloc}/env/gitTools.sh
+
+# load scheduler tools
+. ${envloc}/env/schedulerTools.sh
+
+# compile the model
+# note: this relies on the fact that daint_gnu and daint_intel are the first
+#       two options for the configure script
+echo "### compile model"
+
+compiler_number=0
+case $compiler in
+  gnu) compiler_number=1 ;;
+  intel) compiler_number=2 ;;
+esac
+cd ${rootdir}/FV3
+echo "${compiler_number}" | ./configure
+
+module unload cudatoolkit
+module unload Boost
+module unload cray-python
```

Comment:
is this just to ensure the environment doesn't have too many things loaded that the fortran doesn't need?